### PR TITLE
Enhance package date display

### PR DESCRIPTION
**Pull request by KN4CK3R**
before
grafik

after
grafik

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: templates/package/view.tmpl [hunk 1]
```diff
@@ -39,7 +39,7 @@
 							{{if .HasRepositoryAccess}}
 							<div class="item">{{svg "octicon-repo" 16 "mr-3"}} <a href="{{.PackageDescriptor.Repository.HTMLURL}}">{{.PackageDescriptor.Repository.FullName}}</a></div>
 							{{end}}
-							<div class="item">{{svg "octicon-calendar" 16 "mr-3"}} {{.PackageDescriptor.Version.CreatedUnix.FormatDate}}</div>
+							<div class="item time">{{svg "octicon-calendar" 16 "mr-3"}} {{TimeSinceUnix .PackageDescriptor.Version.CreatedUnix $.locale}}</div>
```
Comment (silverwind) [inline templates/package/view.tmpl]: What's the purpose of the `time` class?

Comment (delvh) [reply templates/package/view.tmpl]: I would have guessed so that the tooltip will be shown?
I don't think `TimeSinceUnix` creates that.

Comment (silverwind) [reply templates/package/view.tmpl]: `TimeSinceUnix` returns `<span class="time-since tooltip" data-content="%s">%s</span>`, it's enough on its own.

Comment (delvh) [reply templates/package/view.tmpl]: oh. I thought it simply converts the number to a human-readable string…
Then why do I find a few of those instances where the declaring tag has `class="time">{{TimeSinceUnix …}}`?

Comment (delvh) [reply templates/package/view.tmpl]: Apparently there is a `time` style class. (`ag '\.time[^a-zA-Z0-9]' web_src/`)

Comment (silverwind) [reply templates/package/view.tmpl]: Those `.time` selectors are not matching here I think.

BTW, I recommend [`rg`](https://github.com/BurntSushi/ripgrep) over `ag`, it honors `.ignore` in this case, as opposed to `ag` which doesn't, so `rg '\.time[^a-zA-Z0-9]' web_src` gives less trash results. 😉

Comment (KN4CK3R) [reply templates/package/view.tmpl]: I thought the class was necessary too...

Comment (KN4CK3R) [inline templates/package/view.tmpl]:
```suggestion
							<div class="item">{{svg "octicon-calendar" 16 "mr-3"}} {{TimeSinceUnix .PackageDescriptor.Version.CreatedUnix $.locale}}</div>
```

**Review by silverwind — APPROVED** [review]

**Pull request merged**